README.md

```diff
@@ -3,12 +3,20 @@ A version of the `<label>` element that works with custom elements as well as na
 
 All text in the `iron-label` will be applied to the target element as a screen-reader accessible description.
 
-There are two ways to use `iron-label` to target an element:
+There are three ways to use `iron-label` to target an element:
 
-1. place an element inside iron-label with the `for` attribute:
+1. place an element inside iron-label and some text:
 
         <iron-label>
-          Context for the Button
+          Label for the Button
+          <paper-button>button</paper-button>
+        </iron-label>
+
+2. place some elements inside iron-label and label one with the `for` attribute.
+The other elements will provide the label for that element:
+
+        <iron-label>
+          <span>Label for the Button</span>
           <paper-button for>button</paper-button>
```

Comment:
Just to get this on record, i think this `for` is really weird, but I also don't have better suggestions :(